### PR TITLE
Feature/styles header section

### PR DESCRIPTION
**Pull request by oleg331**
fix Gulpfile.js
add fonts
making edits from Angelina
add images
add stylesheets for navigation + slider

### CODE REVIEW

File: scss/style.scss [hunk 1]
```diff
@@ -4,4 +4,232 @@
...
+.wrapper{
+}
+
+.header{
+    background: #3d4148;
+}
+
+// btn
+
+.btn{
```
Comment (kobweb) [inline scss/style.scss]: Use separated file for button component

File: scss/style.scss [hunk 2]
```diff
@@ -4,4 +4,232 @@
...
+    display: flex;
+    justify-content: center;
+    margin-bottom: 168px;
+
+    .navigation{
+        display: flex;
+        align-items: center;
+        height: 132px;
+
+        li{
```
Comment (kobweb) [inline scss/style.scss]: Use classes instead of tag selectors. It's related to the whole css

File: scss/style.scss [hunk 3]
```diff
@@ -4,4 +4,232 @@
...
+                .btn{
+                    float: left;
+                    margin-right: 20px;
+                }
+            }
+
+            .slider-arrows{
+
+                .slider-arrow{
+                    display: inline-block;
```
Comment (kobweb) [inline scss/style.scss]: For what `inline-block` here?

File: scss/style.scss [hunk 4]
```diff
@@ -4,4 +4,232 @@
...
+            .slider-dots{
+                display: flex;
+                justify-content: center;
+                position: absolute;
+                left: 0;
+                right: 0;
+                bottom: 34px;
+
+                .slider-dot{
+                    display: inline-block;;
```
Comment (kobweb) [inline scss/style.scss]: For what `inline-block` here? You parent has `display: flex`...

**Review by kobweb — CHANGES_REQUESTED** [review]

**Review by angelinabylitskaya — CHANGES_REQUESTED** [review]
1) use variables for colors, fonts, container width
2) use space before `{` (https://github.com/airbnb/css#rule-declaration)
3) use an empty line before each selector (https://github.com/airbnb/css#formatting)
4) Bad:
 ```
&.active, &:hover{
```
Good:
```
&.active,
&:hover {
```
5) Use empty lines at the end of each file. (Why are you using an empty line at the beginning of a file?)

**Comment by angelinabylitskaya** [comment]
image

**Comment by angelinabylitskaya** [comment]
image
Imports in styles.scss differs from files structure.

**Pull request merged**